release-checklist.md

```diff
@@ -77,8 +79,8 @@ $ git push
 ### 6. Document release
 Document release notes at [https://github.com/tangrams/tangram-es/releases][5].
 
-[1]: https://circleci.com/gh/tangrams/tangram-es
-[2]: https://travis-ci.org/tangrams/tangram-es
-[3]: https://bintray.com/tangrams/maven/tangram
-[4]: https://bintray.com/bintray/jcenter
+[1]: https://app.circleci.com/pipelines/github/tangrams/tangram-es
+[2]: https://oss.sonatype.org/
+[3]: https://central.sonatype.org/pages/ossrh-guide.html
```

Comment:
If I remember this correctly (at least a couple of years since we did this), this has the manual step of "releasing" the artifact right?